src/pudl/transform/ferc1.py

```diff
@@ -1396,6 +1556,74 @@
 "outdoor full" and "outdoor hrsg") are included.
 """
 
+TABLE_UNIT_CONVERSIONS: dict[TABLES_LITERAL, dict] = {
+    "plants_steam_ferc1": {
+        "capex_per_mw": {
+            "column_name_old": "capex_per_kw",
+            "conversion": 1000.0,
+        },
+        "opex_per_mwh": {
+            "column_name_old": "opex_per_kwh",
+            "converstion": 1000.0,
+        },
+        "net_generation_mwh": {
+            "column_name_old": "net_generation_kwh",
+            "converstion": 1000.0,
+        },
+    },
+    "fuel_ferc1": {
+        "fuel_mmbtu_per_unit": {
+            "column_name_old": "fuel_btu_per_unit",
+            "conversion": 1e-06,
+        }
+    },
+}
+"""Table conversions."""
+
+COLUMN_CONDENSING_PRE_STRING_CLEAN: dict[
+    TABLES_LITERAL,
+    dict[str, dict[Literal["keep_records", "remove_records"], dict[str, tuple]]],
+] = {
+    "fuel_ferc1": {
+        "fuel_mmbtu_per_kwh": {
+            "keep_records": (
+                [
+                    "Columbia 1",
+                    "Columbia 2",
+                    "Columbia Total",
+                    "West Campus",
+                ],
+                [
+                    "Coal",
+                    "Coal",
+                    "Coal",
+                    "Gas",
+                ],
+                ["C002498", "C002498", "C002498", "C002498"],
+                [2021, 2021, 2021, 2021],
+            ),
+            "remove_records": (
+                [
+                    "Columbia 1",
+                    "Columbia 2",
+                    "Columbia Total",
+                    "West Campus",
+                ],
+                [
+                    "Coal & Oil",
+                    "Coal & Oil",
+                    "Coal & Oil",
+                    "Gas & Oil",
+                ],
+                ["C002498", "C002498", "C002498", "C002498"],
+                [2021, 2021, 2021, 2021],
+            ),
+        }
+    }
+}
```

Comment:
I.... am not sure if this is the right setup for this process. this dict is paired with `condense_sets_of_records_with_one_datapoint_in_second_record` (i know its a terrible name plz halp).


I have 100% seen this type of issue in other ferc tables so I wanted to keep it pretty general. In this case, I needed to do this condensing *before* we clean the strings for the `fuel_type_code_pudl` column, so I decided to make a specific pre-string-clean dict.

Reply:
turns out i might delete this whole thing bc this column that I was trying to condense into the "real" columns is not even in the current db schema. it seems like a new column. maybe we'll add it even though it's a little bit contrived. 

Also there is reference to this column (`fuel_mmbtu_per_kwh`) in the `REMOVE BAD DATA` of the `fuel_old` function... but i can't find it... halp?